src/services/StatementBuilder.ts

```diff
@@ -0,0 +1,135 @@
+import {HttpVerb} from "./http-verbs";
+import {Statement} from "aws-lambda";
+import {arnToString} from "./resource-arn";
+import {getEnvVar} from "./env-utils";
+
+export type Effect = 'Allow' | 'Deny';
```

Comment:
That's something I usually do (i tend to shove everything into a constant.ts file) but it is also great to have that 'scoped' to the file itself. I am not sure we have guidelines/standards around that from DVSA but it is a nice approach